### ISSUE TITLE
OpenWeatherMap API 3.0 available and call to v2.5 returns HTTP 401

### ISSUE THREAD

**Issue by twnaing** [awersching/wedder#12]
OWM adopts [API v3](https://openweathermap.org/api/one-call-3) and onecall is no longer free. 

The call to v2.5 returns the following message

```
00:44:42 [DEBUG] wedder - Pulling current weather...
00:44:42 [DEBUG] wedder::http - Querying http://api.openweathermap.org/data/2.5/onecall?exclude=minutely,alerts&lat=16.81&lon=96.1512&APPID=[REDACTED] ...
00:44:43 [DEBUG] wedder::http - HTTP 401 Unauthorized
Invalid/unauthorized API key
```

OWM still provides the [current weather](https://openweathermap.org/current) for free tier.

This API point returns <http://api.openweathermap.org/data/2.5/weather?lat=16.81&lon=96.1512&appid=[REDACTED]> data (in json).

EDIT 1: The return data of two APIs are not the same. So changing URL only [at this line](https://github.com/awersching/wedder/blob/0ba239dc432a9529e5db3768efbacd33755f932d/src/weather/owm.rs#L161) is not sufficient.

**Comment by nylocx** [comment]
I stumbled on this too, would be nice to adopt the 2.5/weather endpoint instead of onecall for OpenWeatherMap.

**Comment by awersching** [comment]
Thanks for the hint, it's fixed with c15f7d2554f5765b8943b86ca7c35329787ad202.
Because the current weather API has fewer features, dew point, precipitation chance, and UVI are no longer available.
